app/Models/Member.php

```diff
@@ -63,7 +63,7 @@ public static function Validate_rule($input, $rules, $messages)
      */
     public static function listMember()
     {
-        $datas = Member::all();
+        $datas = Member::where('status', '=', 1)->get();
```

Comment:
A better name for this variable would be $members. Data is already a plural, so datas does not make much sense.